README.md

```diff
@@ -250,22 +253,22 @@ Graphics abstraction:
 
 ## Miscellaneous
 
-- [How HTTPS Works](https://howhttps.works/)
-- [How DNS Works](https://howdns.works/)
-- [Stanford CS Education Library](http://cslibrary.stanford.edu/)
-- [Git.WTF](https://git.wtf/)
-- [Oh shit, git!](https://ohshitgit.com/)
-- [OpenStax Textbooks](https://cnx.org/)
-- [Computer-Science-Resources](https://github.com/the-akira/Computer-Science-Resources.git)
-- [every-programmer-should-know](https://github.com/mtdvio/every-programmer-should-know)
-- [97 Things Every Programmer Should Know](https://97-things-every-x-should-know.gitbooks.io/97-things-every-programmer-should-know/en/)
-
+* [How HTTPS Works](https://howhttps.works/)
+* [How DNS Works](https://howdns.works/)
+* [Stanford CS Education Library](http://cslibrary.stanford.edu/)
+* [Git.WTF](https://git.wtf/)
+* [Oh shit, git!](https://ohshitgit.com/)
+* [OpenStax Textbooks](https://cnx.org/)
+* [Computer-Science-Resources](https://github.com/the-akira/Computer-Science-Resources.git)
+* [every-programmer-should-know](https://github.com/mtdvio/every-programmer-should-know)
+* [97 Things Every Programmer Should Know](https://97-things-every-x-should-know.gitbooks.io/97-things-every-programmer-should-know/en/)
+* [Github Student Education package](https://education.github.com)
```

Comment:
Looks good enough but generally speaking PRs should be specific. And this one touched upon way too much stuff than what you described in the PR.